datarockets-style.gemspec

```diff
@@ -32,10 +32,10 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubocop", "~> 1.56.3"
+  spec.add_dependency "rubocop", "~> 1.61"
```

Comment:
Why not to upgrade to the latest version since we're already upgrading dependencies?